client/src/screens/UnconfiguredScreen.tsx

```diff
@@ -27,11 +53,17 @@ const UnconfiguredScreen = () => {
 
   const [isUploading, setIsUploading] = useState(false)
 
+  const [inputConversionFiles, setInputConversionFiles] = useState<VxFile[]>([])
+  const [isLoading, setIsLoading] = useState(true)
   const [vxElectionFileIsInvalid, setVxElectionFileIsInvalid] = useState(false)
+  const [client] = useState(new ConverterClient('election'))
+  const [isConvertSEMS, setIsConvertSEMS] = useState(false)
```

Comment:
can we use language that is not `SEMS` specific? Unless we agree that SEMS is just "state election management system" and it's not specific to Mississippi. It could be any state converter.

Reply:
Yeah, thought about this. Decided to just do it this way for now until we decide how this interacts with other converters or the absense of converters.